ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -121,3 +121,29 @@ executable db-analyser
                        -threaded
                        -rtsopts
                        "-with-rtsopts=-T -I0 -N2 -A16m"
+
+executable ledger-db-backends-checker
```

Comment:
Did we document this in the readme or somewhere?

Reply:
Not AFAIK, we should add it to `ouroboros-consensus-cardano/README.md`